src/Middleware/RateLimiting/src/RateLimitingMetrics.cs

```diff
@@ -94,13 +94,20 @@ private void LeaseEndCore(in MetricsContext metricsContext, long startTimestamp,
 
         if (metricsContext.CurrentLeasedRequestsCounterEnabled)
         {
-            _currentLeasedRequestsCounter.Add(-1, tags);
+            _activeRequestLeasesCounter.Add(-1, tags);
         }
 
-        if (_leasedRequestDurationCounter.Enabled)
+        if (_requestLeaseDurationCounter.Enabled)
         {
             var duration = Stopwatch.GetElapsedTime(startTimestamp, currentTimestamp);
-            _leasedRequestDurationCounter.Record(duration.TotalSeconds, tags);
+            _requestLeaseDurationCounter.Record(duration.TotalSeconds, tags);
+        }
+
+        if (_requestsCounter.Enabled)
+        {
+            // This modifies the shared tags list so must be the last usage in the method.
+            tags.Add("aspnet.rate_limiting.result", "acquired");
```

Comment:
we can avoid reporting this attribute for the happy case  - see https://github.com/lmolkova/semantic-conventions/pull/1/files#r1279968652

Reply:
I'd like to keep "acquired" to make it obvious what has happened